.githooks/pre-commit

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+#set -x
+PREFIX="pre-commit:"
+RESULTS_FILE=tfsec-scan-output.txt
+TFSEC_RESULTS_DIR=/tmp/tfsec-results
+TFSEC_WORKING_DIR=/tmp/tfsec-files
+
+initialise() {
+    mkdir -p $TFSEC_WORKING_DIR
+    mkdir -p $TFSEC_RESULTS_DIR
+    rm  $TFSEC_WORKING_DIR/* 2> /dev/null
+    rm  $TFSEC_RESULTS_DIR/* 2> /dev/null
+    ALLOW_TFSEC_ISSUES="no"
+}
+
+populateGlobalVars() {
+    stagedFileList=$(git diff --diff-filter=d --cached --name-only)
```

Comment:
All variables should be upper case.

It's generally a good idea to quote VAR="$(some command)"

Reply:
ok